src/NUnitTestExtractor/NUnitTestExtractor/Program.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommandLine;
+using System.IO;
+
+
+namespace NUnitTestExtractor
+{
+    class Program
+    {
+        private static Options _options = new Options();
+
+        private enum Level { Namespace, Class, Function, Null }
+
+        static void Main(string[] args)
+        {
+            //Setting a default value
+            Level level = Level.Null;
+
+            using (var parser = new CommandLine.Parser(with => with.HelpWriter = Console.Error))
+            {
+                if (parser.ParseArgumentsStrict(args, _options, () => Environment.Exit(-1)))
+                {
+                    if (_options.Help)
+                    {
+                        Console.WriteLine(CommandLine.Text.HelpText.AutoBuild(_options));
+                        return;
+                    }
+                   
+                    if(_options.Output != null)
+                    {
+                        if (!Directory.Exists(_options.Output))
+                        {
+                            Directory.CreateDirectory(_options.Output);
+                            Console.WriteLine("created directory at {0}", _options.Output);
```

Comment:
Remove this WriteLine.  The only thing that should be printed to stdout are the "DLL | namespace.class.function" lines (if no output file was specified).